Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs

```diff
@@ -30,5 +31,20 @@ public class SentNotification : BaseNotification
         /// Gets or sets the number of recipients who were throttled out.
         /// </summary>
         public int Throttled { get; set; }
+
+        /// <summary>
+        /// Gets or sets Teams audience id collection.
+        /// </summary>
+        public IEnumerable<string> TeamNames { get; set; }
+
+        /// <summary>
+        /// Gets or sets Rosters audience id collection.
+        /// </summary>
+        public IEnumerable<string> RosterNames { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether a notification should be sent to all the users.
```

Comment:
Change to "... a notification was set to all"

Reply:
Addressed.